src/components/site-wrapper/site-wrapper.tsx

```diff
@@ -10,6 +13,14 @@ export const SiteWrapper = ({ children }: React.PropsWithChildren) => {
             <main className={styles.main}>{children}</main>
             <Footer />
 
+            <Toaster
+                containerStyle={{
+                    inset: '0',
+                }}
+                containerClassName={styles.toaster}
```

Comment:
'react-hot-toast' sets `inset` for toaster '16px' as inline style. I cancel it by inline style, but use class with `padding` to place toasts in a specific position. I need to use classes to keep it responsive (desktop 100px from top, tablet and mobile 0 from top).